apis/python/test/test_ingestion.py

```diff
@@ -245,9 +245,17 @@ def test_ivf_pq_ingestion_u8(tmp_path):
         queries,
         k=k,
         nprobe=nprobe,
-        use_nuv_implementation=True,
     )
-    assert accuracy(result, gt_i) > MINIMUM_ACCURACY
+    query_accuracy = accuracy(result, gt_i)
+    assert query_accuracy > MINIMUM_ACCURACY and query_accuracy < 0.9
+
+    _, result = index_ram.query(
+        queries,
+        k=k,
+        k_factor=2,
+        nprobe=nprobe,
+    )
+    assert accuracy(result, gt_i) > 0.98
```

Comment:
The check of `k_factor=1 -> query_accuracy < 0.9` and `k_factor=2 -> query_accuracy > 0.98` is a bit brittle. I would suggest to just compare the 2 accuracies with each other and assert an improvement threshold.

Reply:
Sure, not totally convinced it's any less brittle 😅, but updated.